float64[3] attitude
float64[3] attitude_rate
float64[3] linear_accel
float64[8] RC
float64[8] RC_Scaled
float64[4] esc_rpm
float64[4] esc_current
float64[4] thrust_att_rates_sp
